background/message-cache.js

```diff
@@ -42,8 +42,8 @@ export async function updateBadge(defaultStoreId) {
         const text = isLoggedIn ? String(count) : "?";
         // The badge will show incorrect message count in other auth contexts.
         // Blocked on Chrome implementing store ID-based tab query
-        await promisify(chrome.browserAction.setBadgeBackgroundColor.bind(chrome.browserAction))({ color });
-        await promisify(chrome.browserAction.setBadgeText.bind(chrome.browserAction))({ text });
+        await promisify(chrome.action.setBadgeBackgroundColor.bind(chrome.action))({ color });
+        await promisify(chrome.action.setBadgeText.bind(chrome.action))({ text });
```

Comment:
Note to self: most calls to `promisify()` are unnecessary on MV3, because Chrome promisified the APIs (Firefox already did it previously, with MV2 support)

Reply:
@TheColaber Promisify() is used in more places